chia/rpc/farmer_rpc_api.py

```diff
@@ -218,23 +218,23 @@ async def _state_changed(self, change: str, change_data: Optional[Dict[str, Any]
         return payloads
 
     async def get_signage_point(self, request: Dict[str, Any]) -> EndpointResult:
-        sp_hash = hexstr_to_bytes(request["sp_hash"])
-        for _, sps in self.service.sps.items():
-            for sp in sps:
-                if sp.challenge_chain_sp == sp_hash:
-                    pospaces = self.service.proofs_of_space.get(sp.challenge_chain_sp, [])
-                    return {
-                        "signage_point": {
-                            "challenge_hash": sp.challenge_hash,
-                            "challenge_chain_sp": sp.challenge_chain_sp,
-                            "reward_chain_sp": sp.reward_chain_sp,
-                            "difficulty": sp.difficulty,
-                            "sub_slot_iters": sp.sub_slot_iters,
-                            "signage_point_index": sp.signage_point_index,
-                        },
-                        "proofs": pospaces,
-                    }
-        raise ValueError(f"Signage point {sp_hash.hex()} not found")
+        sp_hash = bytes32.from_hexstr(request["sp_hash"])
```

Comment:
`hexstr_to_bytes()` supports stripping an optional `0x` prefix from the string as well. This change removes that feature

Reply:
Hm? `from_hexstr()` also supports stripping the optional prefixes.

Reply:
so this code is pointless then? https://github.com/Chia-Network/chia-blockchain/blob/main/chia/util/byte_types.py#L14

Reply:
For `bytes` you use `hexstr_to_bytes()`.  When I worked on the fixed width bytes I made a method for those types, `.from_hexstr()`.